sky/backends/backend_utils.py

```diff
@@ -2797,32 +2797,30 @@ def get_backend_from_handle(
     return backend
 
 
-def get_task_demands_dict(task: 'task_lib.Task') -> Optional[Dict[str, float]]:
-    """Returns the accelerator dict of the task"""
+def get_task_demands_dict(task: 'task_lib.Task') -> Dict[str, float]:
+    """Returns the resources dict of the task"""
     # TODO: CPU and other memory resources are not supported yet.
```

Comment:
update this comment as we do specify cpu below. something like "not supported except for serve controller task"